### PR TITLE
SF6 - Remove deprecated code from TwigBundle

### LINKED ISSUE #34005: SF6 - Remove deprecated code from TwigBundle
## Description 

https://github.com/symfony/symfony/blob/6.4/UPGRADE-6.0.md#twigbundle

EPIC : https://github.com/PrestaShop/PrestaShop/issues/33993

## Acceptance test 

ISO

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Twig service is now private in sf 6
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Test UI and CI are green
| UI Tests          | https://github.com/M0rgan01/ga.tests.ui.pr/actions/runs/7128788665
| Fixed issue or discussion?     | Fixes #34005 
| Related PRs       | -
| Sponsor company   | -

### CODE REVIEW

**Review by boherm — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/config/services/bundle/twig.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ services:
       $environment: "%kernel.environment%"
 
   PrestaShopBundle\Twig\TranslationsExtension:
-    arguments: [ "@service_container", "@router" ]
+    arguments: [ "@router" ]
```
Comment (FabienPapet) [inline src/PrestaShopBundle/Resources/config/services/bundle/twig.yml]: autowire: true

**Review by FabienPapet — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/config/services/bundle/twig.yml (hunk 1 above)
Comment (M0rgan01) [reply src/PrestaShopBundle/Resources/config/services/bundle/twig.yml]: redundant thanks to _defaults.

**Review by boherm — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]
Thanks @M0rgan01

**Comment by paulnoelcholot** [comment]
Hello @M0rgan01,

2 test failling in the CI : https://github.com/paulnoelcholot/testing_pr/actions/runs/7126989453

**Comment by jolelievre** [comment]
Thanks @M0rgan01

**Pull request merged**